### ISSUE TITLE
Issue with controls when using multiple videos

### ISSUE THREAD

**Issue by thiagupct** [Selz/plyr#257]
**Youtube Video 1**:
 `<div class="plyr" data-plyr='{"controls": ["play-large", "play", "progress", "current-time", "mute", "volume", "captions", "fullscreen"], "clickToPlay": "true"}'>`

**Youtube Video 2:** 
`<div class="plyr" data-plyr='{"controls": "[]", "clickToPlay": "false", "loadSprite": "false"}'>`

When loaded _plyr_ accepts controls from video 2 and loads same for the two videos. How can we provide different controls for different videos. Any help is much appreciated!!

**Comment by SamPotts** [comment]
Did you try this?

```html
<div class="plyr" data-plyr='{"controls": [], "clickToPlay": "false", "loadSprite": "false"}'>
```

**Comment by thiagupct** [comment]
@SamPotts Thanks, no it wasn't. Any other solution

**Comment by heinhoang** [comment]
I meet the same problem

**Comment by SamPotts** [comment]
Fixed in v1.7.0
